src/main/resources/data/tconstruct/scripts/tconstruct/tinker_station.zs

```diff
@@ -1,32 +1,78 @@
 
 <recipetype:tconstruct:tinker_station>.addOverslimeModifierRecipe("overslime_test", <item:minecraft:dirt>, 800);
-<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_test", [<item:minecraft:stick> * 3], <item:tconstruct:pickaxe>, "tconstruct:diamond", 2, 5, 1, 1);
-<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_test_requirements", [<item:minecraft:apple> * 3], <tag:items:tconstruct:modifiable>, "tconstruct:emerald", 2, 5, 1, 1, {"tconstruct:haste": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
-<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("inc_modifier_test", <item:minecraft:arrow>, 2, 80, <tag:items:tconstruct:modifiable>, "tconstruct:emerald", 1, 5, 0, 1, <item:minecraft:glass>);
-<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("inc_modifier_test_requirements", <item:minecraft:glowstone>, 1, 4, <tag:items:tconstruct:modifiable>, "tconstruct:haste", 1, 2, 1, 0, <item:minecraft:diamond>, {"tconstruct:emerald": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_example", [<item:minecraft:stick> * 3], <item:tconstruct:pickaxe>, "tconstruct:experienced", 1, 3);
+<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_requirements_example", [<item:minecraft:apple> * 3], <tag:items:tconstruct:modifiable>, "tconstruct:sturdy", 1, 3, {"tconstruct:haste": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addUpgradeModifierRecipe("upgrade_modifier_example", [<item:minecraft:red_sand>], <tag:items:tconstruct:modifiable>, "tconstruct:tasty", 1, 3, 1);
+<recipetype:tconstruct:tinker_station>.addUpgradeModifierRecipe("upgrade_modifier_requirements_example", [<item:minecraft:white_wool>], <tag:items:tconstruct:modifiable>, "tconstruct:reach", 1, 3, 2, {"tconstruct:haste": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addAbilityModifierRecipe("ability_modifier_example", [<item:minecraft:gravel>], <tag:items:tconstruct:modifiable>, "tconstruct:smite", 1, 3, 1);
+<recipetype:tconstruct:tinker_station>.addAbilityModifierRecipe("ability_modifier_requirements_example", [<item:minecraft:orange_wool>], <tag:items:tconstruct:modifiable>, "tconstruct:sharpness", 1, 3, 1, {"tconstruct:emerald": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("incremental_modifier_example", <item:minecraft:black_wool>, 2, 8, <tag:items:tconstruct:modifiable>, "tconstruct:haste", 1, 8, <item:minecraft:black_dye>);
+<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("incremental_modifier_requirements_example", <item:minecraft:yellow_wool>, 3, 8, <tag:items:tconstruct:modifiable>, "tconstruct:searing", 1, 8, <item:minecraft:yellow_dye>, {"tconstruct:emerald": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
```

Comment:
Incremental recipes only really work on certain modifiers. Requires some additional code added in the modifier logic, so most don't support it at this time. Probably better for examples to use existing incremental modifiers.